pkg/store/proxy.go

```diff
@@ -383,78 +393,79 @@ func startStreamSeriesSet(
 				emptyStreamResponses.Inc()
 			}
 		}()
+
+		rCh := make(chan *recvResponse)
+		recvCancel := make(chan bool)
```

Comment:
```suggestion
		done := make(chan struct{})
```

I think this is more common for cancelation channels, more info -> https://blog.golang.org/pipelines#TOC_6.

Reply:
fixed